src/client/cockpit-client-ssh

```diff
@@ -47,8 +47,8 @@ def base64_decode(s):
 
 def cockpit_client_ssh():
     # this is pretty evil, but oh well...
-    xdg_data_home = os.getenv('XDG_DATA_HOME') or os.path.expanduser('~/.local/share')
-    askpass = os.path.join(xdg_data_home, 'cockpit-client-askpass')
+    xdg_data_home = os.getenv('XDG_CACHE_HOME') or os.path.expanduser('~/.cache')
+    askpass = os.path.join(xdg_data_home, 'cockpit-client-ssh-askpass')
```

Comment:
eh....

I think we should aim to get rid of `cockpit-ssh-askpass`, and soon.  For me, the point of having `cockpit-beiboot` in beta in flatpak already is that we gain a bit of confidence with just enabling this stuff by default.

I don't enjoy having 3 copies of cockpit-ssh in our tree...

Reply:
I see, I sort of was under the impression that cockpit-client calls one or the other based on the availability of cockpit-bridge. But I checked the code, and c-beiboot itself handles the "existing bridge" case, so I'll change it to that instead.

Reply:
As long as we ship it, I'd like to keep the test case. It's cheap to have. But I'll rename it to testClientSSH, we can then drop it together with c-client-ssh.

Reply:
Updated as discussed above.